### PR TITLE
Rename UG AB3 References

### LINKED ISSUE #128: Update UG to reflect Transaction commands
https://ay2425s1-cs2103-f13-1.github.io/tp/UserGuide.html#features

### LINKED ISSUE #132: [ONGOING] add txn related UG

### LINKED ISSUE #192: Update the User Guide for AddTxn
Update User Guide for AddTxn to include new changes

- Include documentation to be able to add categories
- Replace documentation for command input from phone number to index

### LINKED ISSUE #206: UG: Rename all references from "AB3" to "SpleetWaise"
We should update all instances of "AB3" in the UG to "SpleetWaise".  For E.g:

Screenshot 2024-10-29 at 9 49 57 PM

### PR DESCRIPTION
**Pull request by SeeYangZhi**
Closes #206, Closes #128, Closes #132, Closes #192

## Summary:

This pull request updates the User Guide documentation to reflect changes from AddressBook Level 3 (AB3) to SpleetWaise. The modifications include renaming references to the application, updating descriptions to match the new functionality of SpleetWaise, and adjusting command examples to align with the updated application features.

## Changes Made:

- Updated the introductory section to describe SpleetWaise's purpose, emphasising its focus on expense tracking for students.
- Changed download instructions to point to the new repository URL for SpleetWaise.
- Updated command examples and descriptions to reflect the new application name and functionality.
- Added tips for understanding transaction amounts and their implications.
- Clarified data saving and editing instructions, including the introduction of a new JSON file for transaction data.
- Revised the FAQ section to reference SpleetWaise instead of AddressBook.
- Updated the command summary for transactions to match the new command syntax and examples.

## Motivation:

The changes are necessary to ensure that the User Guide accurately reflects the new application's branding and functionality, providing users with clear and relevant instructions for using SpleetWaise effectively.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -343,8 +345,7 @@ _Details coming soon ..._
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains
-the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous SpleetWaise home folder.
```
Comment (rollingpencil) [inline docs/UserGuide.md:348]:
```suggestion
**A**: Install the app in the other computer and overwrite the sample data file it creates with the file that contains the data of your previous SpleetWaise home folder.
```

**Review by rollingpencil — DISMISSED** [review]
LGTM! Just a typo.

File: docs/UserGuide.md (hunk 1 above)
Comment (SeeYangZhi) [reply docs/UserGuide.md:348]: I believe when we just install the app, there isnt even the data file. we should test out, if we copy over the data files to the new app location, it should work accordingly.

**Review by rollingpencil — APPROVED** [review]
LGTM!

**Pull request merged**